sdk/textanalytics/azure-ai-textanalytics/src/samples/README.md

```diff
@@ -55,8 +63,12 @@ Batch Samples:
 - [Extract key phrases in a batch of documents with asynchronous client(Convenience)][async_sample_key_phrases_batch_convenience]
 - [Analyze sentiment in a batch of documents][sample_sentiment_batch]
 - [Analyze sentiment in a batch of documents(Convenience)][sample_sentiment_batch_convenience]
+- [Analyze sentiment with opinion mining in a batch of documents][sample_sentiment_opinion_mining_batch]
+- [Analyze sentiment with opinion mining in a batch of documents(Convenience)][sample_sentiment_opinion_mining_batch_convenience]
 - [Analyze sentiment in a batch of documents with asynchronous client][async_sample_sentiment_batch]
 - [Analyze sentiment in a batch of documents with asynchronous client(Convenience)][async_sample_sentiment_batch_convenience]
+- [Analyze sentiment with opinion mining in a batch of documents with asynchronous client][async_sample_sentiment_opinion_mining_batch]
+- [Analyze sentiment with opinion mining in a batch of documents with asynchronous client(Convenience)][async_sample_sentiment_opinion_mining_batch_convenience]
```

Comment:
Do you think there is value on showing this for batch too? nothing changes here, so it is the same as the single document sample.

Reply:
Do you mean there is no difference between 'batch operation with opinion mining' and 'batch operation without opinion mining'?

Reply:
Correct. What changes is the wrapper around documents, but not OM as a core, so not point showing it. no?

Reply:
Fair point. I can remove it.